drivers/serial/uart_mcux.c

```diff
@@ -401,6 +403,17 @@ static const struct uart_mcux_config uart_mcux_##n##_config = {		\
 }
 
 #ifdef CONFIG_UART_INTERRUPT_DRIVEN
+#if DT_INST_PROP(0, no_error_irq)
```

Comment:
you can't just check instance 0, look into COND_CODE_0 macro

Reply:
You mean replace it with this condition?
`#if COND_CODE_0(no_error_irq, false, true)`

Reply:
no, the cond_code_0 macro can be used inside other macros, so you can use it to check this for every uart instance rather than just instance 0.

Reply:
updated